src/code_80057C60.c

```diff
@@ -742,7 +742,7 @@ void func_800591B4(void) {
                         }
                         func_8004F3E4(0);
                     }
-                    if ((D_800DC530 == 1) && (D_80165801 != 0)) {
+                    if ((gScreenModeSelection == SCREEN_MODE_2P_UD) && (D_80165801 != 0)) {
```

Comment:
Avoid abbreviations such as UD. Using full/long names makes it so people always know what the meaning is. What's your philosophy behind UD/LR. Vs perhaps another way of writing it?

For instance why not `2PlayerMode`?

And just for clarification. My goal here is just to help us have the best names possible. so that someone not fluent in mk64 code can understand it. I'm not criticizing your approach, just helps me understand.

Reply:
No worries, happy to get feedback since I would like this to be understandable. UD/LR is from the text on the debug menu, but I think you're right that it's worth spelling out up/down and left/right. Same for "Player" instead of "P"

Reply:
I suppose this is worth having a debate over... I kinda think `SPLITSCREEN_HORIZONTAL` `SPLITSCREEN_VERTICAL`. Would maaaaybe be better. But there is an argument to be had, from a decomp perspective to use the in-game names.

Reply:
I'm fine with P1, P2, P3, P4 for flags. I'm pretty sure everyone will understand it. But if you're going to do that. Just make sure you use that in-places that make sense.

Reply:
I like adding `SPLITSCREEN`, I think it makes it a bit clearer what `SCREEN_MODE` is actually about. 

So `SCREEN_MODE_1P`, `SCREEN_MODE_2P_SPLITSCREEN_HORIZONTAL`, `SCREEN_MODE_2P_SPLITSCREEN_VERTICAL`, `SCREEN_MODE_3P_4P_SPLITSCREEN`?

Reply:
Yea, I think those work great.